services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/assign_a_diagnostic.tsx

```diff
@@ -43,6 +49,10 @@ const STARTER_POST_TEST_LOCKED_TEXT = "This is locked because you haven't assign
 const INTERMEDIATE_POST_TEST_LOCKED_TEXT = "This is locked because you haven't assigned the Intermediate Baseline Diagnostic yet. Assign it to unlock the Intermediate Growth Diagnostic."
 const ADVANCED_POST_TEST_LOCKED_TEXT = "This is locked because you haven't assigned the Advanced Baseline Diagnostic yet. Assign it to unlock the Advanced Growth Diagnostic."
 
+const ELL_STARTER_POST_TEST_LOCKED_TEXT = "This is locked because you haven't assigned the ELL Starter Baseline Diagnostic yet. Assign it to unlock the ELL Starter Growth Diagnostic."
+const ELL_INTERMEDIATE_POST_TEST_LOCKED_TEXT = "This is locked because you haven't assigned the ELL Intermediate Baseline Diagnostic yet. Assign it to unlock the ELL Intermediate Growth Diagnostic."
+const ELL_ADVANCED_POST_TEST_LOCKED_TEXT = "This is locked because you haven't assigned the ELL Advanced Baseline Diagnostic yet. Assign it to unlock the ELL Advanced Growth Diagnostic."
+
```

Comment:
I'm wondering if we should just eventually pull all of these constants out into the `assignmentFlowConstants` file and also add constants for the 'What' and 'When' text there as well

Reply:
Yeah, I think you're right, it's time for a refactor here. I'll make a follow-up card to do that.